src/nyc_trees/apps/login/forms.py

```diff
@@ -39,6 +40,12 @@ class NycRegistrationForm(RegistrationFormUniqueEmail):
         required=False
     )
 
+    def clean_username(self):
+        if self.cleaned_data['username'].lower() in User.reserved_usernames:
+            raise ValidationError('This username is invalid because it '
```

Comment:
This is a little specific and jargony for for a user facing error. "This username is already in use" is a white lie, but, perhaps, a bit friendlier.